config/src/main/java/com/networknt/config/ConfigInjection.java

```diff
@@ -160,4 +169,29 @@ public void setKey(String key) {
             this.key = key;
         }
     }
+
+    // Method used to cast string into int, double or boolean
+    private static Object typeCast(String str) {
+        if (str == null || str.equals("")) {
+            return null;
+        }
+        if (isBoolean(str)) {
+            return str.equals("true") || str.equals("True") || str.equals("TRUE") ? true : false;
+        }
```

Comment:
Same comment from that other pr, I think this can be simplified

Reply:
Yes, I will simplify it.

Reply:
@jiachen1120 Should the boolean casting enhancement be submitted to this branch too? Maybe you can specify the merge order your expecting between all the config enhancement prs..

Reply:
Hey, sorry for making the PR kaos... The merge order should be:
**[1]** https://github.com/networknt/light-4j/pull/375
**[2]** https://github.com/networknt/light-4j/pull/386
**[3]** https://github.com/networknt/light-4j/pull/387
**[4]** https://github.com/networknt/light-4j/pull/388

On the other words, 386 make changes on 375, 387 make changes on 386, 388 make changes on 387. So, 388 contains all the comments in term of the config changes. So, in this one, I think it not necessary to submit boolean casting change to this branch.

I will be awared next time to make all the change based on develop to avoid kaos and conflicts. So sorry about make your work confused.

Reply:
We will keep in mind when merging these related issues. I am wondering if we should group related issues into one branch in the future or not. What do you think?

Reply:
We used once for light-codegen the "WIP" branch, and merged commits from @BalloonWen and me, tested all out, ironed out what was required and submitted a single PR to develop.

We might as well use the same WIP for temporary merges

Reply:
One suggestion: 
we redirect all 4 PRs into a PRBranch or WIPBranch, merge them all there, test and review against all outstanding RFC docs.

One PR to develop will bring them all in

Reply:
My opinion is that there is no need to merge them all into a separate branch before merging into develop. Everything from develop should be merged into each feature/fix branch and tested before each feature/fix is merged.